dev/breeze/src/airflow_breeze/utils/docker_command_utils.py

```diff
@@ -381,12 +381,15 @@ def prepare_base_build_command(image_params: CommonBuildParams) -> list[str]:
             ]
         )
         if not image_params.docker_host:
+            builder = get_and_use_docker_context(image_params.builder)
             build_command_param.extend(
                 [
                     "--builder",
-                    get_and_use_docker_context(image_params.builder),
+                    builder,
                 ]
             )
+            if builder != "default":
```

Comment:
When we are using builder different than default we use `--load` flag to load the built image to local docker engine.